gulpfile.js

```diff
@@ -106,9 +106,11 @@ var cssFilePaths = [];
 var jsFilePaths = [];
 var fontFolderPaths = [];
 var cssBackgroundFilepaths = [];
+// There is no cache slug for dev.
+var cacheSlugDev = '';
 var generatedTargetDir = path.join(
   'third_party', 'generated',
-  isMinificationNeeded ? 'prod' : 'dev');
+  isMinificationNeeded ? 'prod' : cacheSlugDev);
```

Comment:
I was thinking about doing this earlier, but now it's not required. Everything is handled by build.py.